tools/shared/src/main/scala/org/scalajs/core/tools/javascript/Trees.scala

```diff
@@ -61,7 +61,12 @@ object Trees {
     def ref(implicit pos: Position): Tree = VarRef(name)
   }
 
-  case class ParamDef(name: Ident)(implicit val pos: Position) extends Tree {
+  /** ES6 let or const (depending on the mutable flag). */
+  case class Let(name: Ident, mutable: Boolean, rhs: Tree)(implicit val pos: Position) extends Tree {
+    def ref(implicit pos: Position): Tree = VarRef(name)
+  }
```

Comment:
Do we still use `ref` anywhere? It seemed to be useful at some point, but now most of the methods return a `Tree` so we construct the `VarRef` manually...